mongo/healthcheck.go

```diff
@@ -2,7 +2,7 @@ package mongo
 
 import (
 	"github.com/ONSdigital/go-ns/log"
-	"github.com/globalsign/mgo"
+	mgo "github.com/gedge/mgo"
```

Comment:
When I attempt to `go get github.com/gedge/mgo` I get the following error:

gedge/mgo/auth.go:38:2: use of internal package not allowed

Reply:
yes, I think that's because I put my changes on a different branch: `feature/mongotimestamp-funcs-n-imports`
to avoid changing all the `import` statements on `master`
not sure how to tell `go get` to get a particular branch